### ISSUE TITLE
Add Fly Output Options for Parallel Tasks in a Job

### ISSUE THREAD

**Issue by reskin89** [concourse/concourse#5403]
## What challenge are you facing?

When tasks run in parallel during a job, when using fly watch to review the output log, it can be difficult if tasks have a lot of output, or there are many, to potentially isolate one you are interested in further investigation (this is not an issue in the GUI).

## What would make this better?

Potentially an addition option to `fly watch` to specify a task name

## Are you interested in implementing this yourself?

Potentially if I can find the capacity to create another PR (I still have one to finish updating :) )

**Comment by taylorsilva** [comment]
# Issue Bankruptcy

This issue is being closed as part of 2025's **Issue Bankruptcy** declaration. [See here for details](https://blog.concourse-ci.org/posts/2025-01-06-concourse-2024-review/)

If you think this issue still has merit then feel free to open a new issue and reference this one.
